Front/addsavenote.cpp

```diff
@@ -147,3 +148,19 @@ void AddSaveNote::on_SaveNoteButton_clicked()
 
 }
 
+
+void AddSaveNote::on_ASNCanselButton_clicked()
+{
+    Category* w = new Category();
```

Comment:
Not deallocated memory, use smart pointer

Reply:
Fixed